AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/test/TestHierarchicalVersionUtils.dfy

```diff
@@ -0,0 +1,49 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+include "../src/HierarchicalVersionUtils.dfy"
+
+module {:options "/functionSyntax:4" } TestHierarchicalVersionUtils {
+  import HVUtils = HierarchicalVersionUtils
+  import Structure = HVUtils.Structure
+
+  const NEWLINE := "\n"
+  const SPACE := " "
+  const TAB := "\t"
+  const SLASH_R := "\r"
+  const BLANKISH: seq<string> := [NEWLINE, SPACE, TAB, SLASH_R]
+  const BLANK := ""
+  const JOHN := "john"
+  const VALID_PREFIXED: seq<string> := [Structure.ENCRYPTION_CONTEXT_PREFIX + JOHN]
+
+  const EMPTYISH_PREFIXED: seq<string> :=
+    seq(|BLANKISH|-1, n requires 0 <= n <= |BLANKISH|-1 => Structure.ENCRYPTION_CONTEXT_PREFIX + BLANKISH[n])
+    + [Structure.ENCRYPTION_CONTEXT_PREFIX + BLANK]
+
+  const TERMINAL_EMPTYISH_PREFIXED: seq<string> :=
+    seq(
+    |BLANKISH|-1,
+    n requires 0 <= n <= |BLANKISH|-1 => Structure.ENCRYPTION_CONTEXT_PREFIX + JOHN + BLANKISH[n])
+
+  const START_EMPTYISH_PREFIX: seq<string> :=
+    seq(
+    |BLANKISH|-1,
+    n requires 0 <= n <= |BLANKISH|-1 => Structure.ENCRYPTION_CONTEXT_PREFIX + BLANKISH[n] + JOHN)
+
+  method {:test} /* {:verify false} */ TestRemovePrefix() {
```

Comment:
```suggestion
  method {:test} TestRemovePrefix() {
```

Reply:
I will fix this another time. Want to get this in for some other testing.